src/ui/libs/DatalensChartkit/ChartKit/plugins/Table/renderer/utils/renderer.tsx

```diff
@@ -1,21 +1,83 @@
 import React from 'react';
 
 import {dateTime} from '@gravity-ui/date-utils';
+import {CaretLeft, CaretRight} from '@gravity-ui/icons';
+import {Icon} from '@gravity-ui/uikit';
+import block from 'bem-cn-lite';
 import get from 'lodash/get';
 import {
     BarTableCell,
     BarViewOptions,
+    ChartKitTableQa,
     NumberViewOptions,
     TableCommonCell,
     TableHead,
     isMarkupItem,
 } from 'shared';
 
+import type {THead} from '../../../../../../../components/Table/types';
 import {numberFormatter} from '../../../../components/Widget/components/Table/utils/misc';
 import {BarCell} from '../components/BarCell/BarCell';
 import {MarkupCell} from '../components/MarkupCell/MarkupCell';
 import {TreeCell} from '../components/TreeCell/TreeCell';
```

Comment:
Could you add `type` keyword for this imports?

Reply:
No, here `TreeCell` is a component, not a type